dbt/models/iasworld/schema/iasworld.comdat.yml

```diff
@@ -49,6 +49,26 @@ sources:
             description: Total business living area
           - name: calcexclude
             description: Exclude this record from income calculations
+          - name: calc_meth
+            description: '{{ doc("shared_column_calc_meth") }}'
+            tests:
+              - accepted_values:
+                  name: iasworld_comdat_calc_meth_accepted_values
```

Comment:
This test, along with the `external_*` tests below, represent the tests Mirella requested to check the values of new calculated columns.